sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -471,7 +471,7 @@ object Qonversion : LifecycleDelegate {
     )
     @JvmStatic
     fun handleNotification(remoteMessage: RemoteMessage) =
-        automationsManager?.handlePushIfPossible(remoteMessage) ?: run {
+        automationsManager?.handlePushIfPossible(remoteMessage.data) ?: run {
             logLaunchErrorForFunctionName(object {}.javaClass.enclosingMethod?.name)
             return@run false
         }
```

Comment:
I'd offered a bit different solution - instead of calling `automationManager` and handling its' errors you can just call overloaded function `handleNotification`:

`fun handleNotification(remoteMessage: RemoteMessage) = handleNotification(remoteMessage.data)`